### ISSUE TITLE
Custom Cosmetic not accurate in-game to unity

### ISSUE THREAD

**Issue by chowder908** [notnotnotswipez/MoreCompany#31]
![image](https://github.com/notnotnotswipez/MoreCompany/assets/12567023/985a7435-1f79-4c5b-8b5f-703edee0d6cd)
as you can see in unity it looks how I want it.

![image](https://github.com/notnotnotswipez/MoreCompany/assets/12567023/48542fbc-e041-4389-b81c-8bba29b165f1)
But in-game it's way off and I have no idea how to fix it....

**Comment by chowder908** [comment]
![image](https://github.com/notnotnotswipez/MoreCompany/assets/12567023/8bea5f97-4b42-4ed1-bc7e-802c34816a1c)
I have tried literally everything from blender to unity and it just refuses to rotate....

**Comment by chowder908** [comment]
![image](https://github.com/notnotnotswipez/MoreCompany/assets/12567023/72f0783d-4c8c-43ac-b64d-42f64eec9193)
I have tried everything in blender and unity it just refuses to rotate

**Comment by notnotnotswipez** [comment]
Are you moving, scaling, or rotating the gameobjects titled “DO NOT MOVE”?

**Comment by chowder908** [comment]
> Are you moving, scaling, or rotating the gameobjects titled “DO NOT MOVE”?

Yes, I did find a a fix by adding a rotational constraint to the model

**Comment by AncientPipebomb** [comment]
> > Are you moving, scaling, or rotating the gameobjects titled “DO NOT MOVE”?
> 
> No I did the model that was placed there underneath that like the tutorial says, but I found a fix by adding a rotational constraint to the model

may i know which moment to put the rotational contraint? (blender or unity) and which model? (cosmetic or scavenger model)
